### DISCUSSION TITLE
[Future idea] Creation of Nonprofit organization

### DISCUSSION THREAD

**Discussion by inson1** [FossifyOrg/General-Discussion#25] [Ideas]
I dont know where are you from. But in most contries Nonprofit organization doesnt pay taxes for donations and donators too. 
Plus we dont want another selling problem. 
It shows that you are commited to non profit open source software.

Its just idea for the future.

**Comment by inson1** [comment]
@jakecharlie

> My suggestion is that in the future the name "FossifyOrg" brand, the logos of each icon created and even the URLs of the F-Droid version and perhaps the Playstore could not be sold again by someone here, as the main person responsible for this new Project.
> 
> Note that I'm not referring to the source code, which can be created as a fork, I'm referring to what can be copyrighted.
> 
> That's why I suggest that lawyers and people with knowledge of law study possibilities such as:
> 
> 501c3 nonprofit in USA (like signal.org)
> Foundation in Switzerland (like Quad9 DNS Foundation)
> -Private Limite Company with Apache License 2.0 (like Organic Maps OÜ, the fork of Maps.me).
> I repeat: There is no point investing time and effort in this new project if in the future the new founder can sell everything without needing votes for it and without needing to satisfy anyone here. He puts the money in his pocket and everything is fine. Please, don't make that mistake again.We shouldn't trust anyone, so there must be legal barriers to prevent what happened with Simple Mobile Tools happen here again in the future.

**Comment by ghost** [comment]
It is important that this idea is taken forward, because at the end of the day, even by creating a new fork, the privacy of the user base will have been sold and this will be the biggest damage caused by a possible sale.

**Comment by naveensingh** [comment]
A non-profit org is the goal. The organization name and the .org domain were chosen for a reason. I had thought about this before I started the fork. A .com might persuade me or future maintainers to commercialize the project but a .org will not. This is also a reason why I don't want to change the project name. PureApps, CleanDroid, CoreApps, and YouApps (existing project) can all be commercialized and nobody would question the name. 

Of course, I could respond with "Trust me, I won't sell the project, I believe in FOSS yada yada" but to me, that's not reassuring. I think we are too complex to really know ourselves or anyone else. It's best if the project doesn't allow such an action by design, not by promise.

With that said, Fossify won't magically become a registered nonprofit after a month, it'll take time and paperwork. As mentioned in the title, this is for the future. For that same reason, I won't be active here but this discussion will stay open until Fossify is registered as a non-profit or dies due to starvation (both are real possibilities).

**Reply by inson1** [reply]
@naveensingh Are you from United States, so we would know what kind of lawyer we need? Or where would the non profit org be set?
Maybe there is someone who can help.

**Reply by naveensingh** [reply]
> Or you from United States, so we would know what kind of lawyer we need? Or where would the non profit org be set?

No. India.
